Error: Cannot find module 'cst'

Hi, after installation I have this error when I try to use `jscs app/`

```
Error: Cannot find module 'cst'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/travis/build/Selleo/NTFrontend/node_modules/ember-suave/node_modules/jscs-ember-deprecations/lib/rules/disallow-emberkeys.js:2:11)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
```

How can I solve it?

--- a/package.json
+++ b/package.json
@@ -28,7 +28,6 @@
     "ember-cli-blueprint-test-helpers": "^0.11.0",
     "ember-cli-internal-test-helpers": "^0.8.1",
     "ember-cli-release": "1.0.0-beta.1",
-    "jscs": "^3.0.4",
     "mocha": "^2.4.5",
     "mocha-jshint": "^2.1.1",
     "walk-sync": "^0.2.6"
@@ -40,11 +39,10 @@
   ],
   "dependencies": {
     "broccoli-jscs": "^2.0.0",
-    "cst": "^0.3.0",
     "ember-cli-babel": "^5.1.6",
     "js-string-escape": "^1.0.0",
     "jscs": "^3.0.4",
-    "jscs-ember-deprecations": "martndemus/jscs-ember-deprecations#ae206b8",
+    "jscs-ember-deprecations": "brzpegasus/jscs-ember-deprecations#5749615",
     "temp": "0.8.3"
   },
   "ember-addon": {

